shacl12-core/index.html

```diff
@@ -5683,6 +5685,98 @@ <h4>sh:qualifiedValueShape, sh:qualifiedMinCount, sh:qualifiedMaxCount</h4>
 						</div>
 					</aside>
 				</section>
+
+				<section id="ReifiableByShapeConstraintComponent">
+					<h4>sh:reifiableBy, sh:reificationRequired</h4>
+					<p>
+						<code>sh:reifiableBy</code> can be used to link a <a>property shape</a> with one or more <a>node shapes</a>.
+						Any reified statement must conform to these node shapes.
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<tr>
+							<th>Property</th>
+							<th>Summary and Syntax Rules</th>
+						</tr>
+						<tr>
+							<td><code>sh:reifiableBy</code></td>
+							<td>
+								The <a>node shape</a> that the reified statement must conform to.
+								<span data-syntax-rule="reifiableBy-node">The values of <code>sh:reifiableBy</code> must be <a>well-formed</a> <a>node shapes</a>.</span>
+							</td>
+						</tr>
+						<tr>
+							<td><code>sh:reificationRequired</code></td>
+							<td>
+								This is an <a>optional parameter</a> of <code>sh:ReifiableByConstraintComponent</code>.
+								If set to <code>true</code> there must be at least one reification value for the focus node/path combination in the <a>data graph</a>.
+								<span data-syntax-rule="reificationRequired-datatype">The values of <code>sh:reificationRequired</code> in a shape are literals with datatype <code>xsd:boolean</code>.</span>
+							</td>
+						</tr>
+					</table>
+
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="ReifiableBy">
+							Let <code>t</code> be the <a>triple terms</a> of the triples obtained from the last traversal step when following the <code>$path</code> <a>SHACL property path</a> of the <a>property shape</a> from the <a>focus node</a> to the <a>value node</a>.
+							For each reified statement for the <a>triple terms</a> <code>t</code>, a failure MUST be produced if the validation of the reified statement with the <a>reifier</a> as <a>focus node</a> against the <a>node shape</a> <code>$reifiableBy</code> produces a <a>failure</a>.
```

Comment:
Note the terminology here: I believe you mean validation result, not failure. Validation result is something like sh:Violation. Failure is used when the validation produces something like an exception.  The same applies in the next text definition.

Reply:
Yes, that would make more sense. I aligned the wording to the other shape-based and logical constraint components. All of them use the wording "produces a failure". Should we also change them?

Reply:
To me the logical components look correct. They basically forward any failure that happens during the evaluation of the nested shapes, but they normally return a validation result (e.g. sh:Violation) when the nodes violate the shape. So I think there is still a sentence missing that a validation result is produced when the validation of the reification does not conform to the given shape.